Source/DataSources/KmlDataSource.js

```diff
@@ -421,14 +421,16 @@ function embedDataUris(div, elementType, attributeName, uriResolver) {
   for (let i = 0; i < elements.length; i++) {
     const element = elements[i];
     const value = element.getAttribute(attributeName);
-    const relativeUri = new Uri(value);
-    const uri = relativeUri.absoluteTo(baseUri).toString();
-    const index = keys.indexOf(uri);
-    if (index !== -1) {
-      const key = keys[index];
-      element.setAttribute(attributeName, uriResolver[key]);
-      if (elementType === "a" && element.getAttribute("download") === null) {
-        element.setAttribute("download", key);
+    if (value !== null) {
```

Comment:
Should this code path only apply to `null` values, or should `undefined` values be handled in the same way? We prefer to [use the Cesium `defined` function to check for both `undefined` and `null` values](https://github.com/CesiumGS/cesium/blob/main/Documentation/Contributors/CodingGuide/README.md#basic-code-construction), and it appears that if a value is provided to the `Uri` constructor, it will throw `TypeError: undefined is not a valid argument for URI` if that value is `undefined`.

Reply:
Thanks for the feedback @ggetz! I have switched to `defined` as it is more consistent with Cesium style.

However, please note that there are other places in the same file where we explicitly test for `null`. I think it is mostly ok as most browsers return `null` for `getAttribute()` calls when the attribute is non-existent.

Reply:
This should be fine for the scope of this PR. Thanks!